skrf/network.py

```diff
@@ -166,7 +166,7 @@
 
 import numpy as npy
 from numpy.linalg import inv as npy_inv
-from numpy import fft, gradient, ndarray, reshape, shape, ones
+from numpy import fft, gradient, ndarray, reshape, shape, ones, std, dstack
```

Comment:
`dstack` seems not use after. 

```suggestion
from numpy import fft, gradient, ndarray, reshape, shape, ones, std
```